File: hammer-gravity/src/main/java/dev/vini2003/hammer/gravity/HG.java
package dev.vini2003.hammer.gravity;

import dev.vini2003.hammer.gravity.registry.common.HGCommands;
import dev.vini2003.hammer.gravity.registry.common.HGEvents;
import dev.vini2003.hammer.gravity.registry.common.HGNetworking;
import net.fabricmc.api.ModInitializer;
import org.jetbrains.annotations.ApiStatus;

@ApiStatus.Internal
public class HG implements ModInitializer {
	@Override
	public void onInitialize() {
		HGCommands.init();
		HGEvents.init();
		HGNetworking.init();
	}
}

File: hammer-gravity/src/main/kotlin/dev/vini2003/hammer/gravity/HG.kt
package dev.vini2003.hammer.gravity

import dev.vini2003.hammer.gravity.registry.common.HGCommands
import dev.vini2003.hammer.gravity.registry.common.HGEvents
import dev.vini2003.hammer.gravity.registry.common.HGNetworking
import net.fabricmc.api.ModInitializer

object HG : ModInitializer {
	override fun onInitialize() {
		HGNetworking.init()
		HGCommands.init()
		HGEvents.init()
	}
}